src/routes/transactions/entities/transaction-history-group.entity.ts

```diff
@@ -0,0 +1,11 @@
+import { MultisigTransaction } from '../../../domain/safe/entities/multisig-transaction.entity';
+import { ModuleTransaction } from '../../../domain/safe/entities/module-transaction.entity';
+import { EthereumTransaction } from '../../../domain/safe/entities/ethereum-transaction.entity';
+
+export class TransactionDomainGroup {
+  timestamp: number;
+  transactions:
+    | MultisigTransaction[]
+    | ModuleTransaction[]
+    | EthereumTransaction[];
+}
```

Comment:
I need this to group the domain transactions, maybe should move it to domain folder. :thinking:

Reply:
This is only used in `TransactionMapper`. Why not have it there?

Reply:
Done 354fbed6e78c83e90cf797088b4d3f5fc3f9bfc4